lib/ui/semantics.dart

```diff
@@ -819,6 +819,7 @@ abstract class SemanticsUpdateBuilder {
     required Int32List childrenInTraversalOrder,
     required Int32List childrenInHitTestOrder,
     required Int32List additionalActions,
+    int? headingLevel,
```

Comment:
Please add dartdocs for this new parameter (only in the mobile version; other platforms use the mobile dartdocs).

Reply:
Ok, I added dartdocs for the parameter.